bool dummy
---
bool success
string dev_list
uint8 error
